<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoWidget</class>
 <widget class="QWidget" name="TodoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>800</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>921</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter a task item</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>130</y>
       <width>101</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icon/toplion图标库 _添加-71.png</normaloff>../../icon/toplion图标库 _添加-71.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>1061</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>640</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/manage.png</normaloff>:/icon/manage.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>640</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/check.png</normaloff>:/icon/check.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>640</y>
       <width>201</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icon/delete2.png</normaloff>:/icon/delete2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="GotoFinishedpageButton">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>640</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">GotoFinishedpage</string>
     </property>
     <property name="text">
      <string>Finished Page</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icon/Chevron-Right 2.png</normaloff>../../icon/Chevron-Right 2.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="listWidget_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>110</y>
       <width>611</width>
       <height>511</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Algerian&quot;;
text-decoration: line-through;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>749</x>
       <y>470</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>170</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Algerian&quot;;</string>
     </property>
     <property name="text">
      <string>Unfinished Tasks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>230</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Algerian&quot;;</string>
     </property>
     <property name="text">
      <string>Finished Tasks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>420</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Write your summary:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>650</y>
       <width>181</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icon/toplion图标库 _更多管理.png</normaloff>../../icon/toplion图标库 _更多管理.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>650</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icon/toplion图标库 _删除.png</normaloff>../../icon/toplion图标库 _删除.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>110</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Algerian&quot;;</string>
     </property>
     <property name="text">
      <string>All Tasks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>749</x>
       <y>280</y>
       <width>531</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Notice: All finished tasks will be auto cleared,&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;try to finish all the task in your day!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="GotoUndonepageButton">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>650</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Undone Page</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icon/Chevron-Left 2.png</normaloff>../../icon/Chevron-Left 2.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1261</width>
     <height>781</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
